packages/notifi-frontend-client/lib/storage/LocalForageStorageDriver.ts

```diff
@@ -26,9 +26,8 @@ const getEnvPrefix = (env: NotifiEnvironment): string => {
 export const createLocalForageStorageDriver = ({
   env,
   dappAddress,
-  walletPublicKey,
 }: NotifiFrontendConfiguration): StorageDriver => {
-  const keyPrefix = `${getEnvPrefix(env)}:${dappAddress}:${walletPublicKey}`;
```

Comment:
ah unfortunately we need to somehow preserve the publickey in the key.

probably means we need to expose a second parameter